test/e2e-tests.sh

```diff
@@ -23,8 +23,8 @@ export PATH=$PATH:${REPO_ROOT_DIR}
 # Will create and delete this namespace (used for all tests, modify if you want a different one used)
 export KN_E2E_NAMESPACE=kne2etests
 
-export KNATIVE_EVENTING_VERSION="0.21.0"
-export KNATIVE_SERVING_VERSION="0.21.0"
+export KNATIVE_EVENTING_VERSION="0.22.0"
+export KNATIVE_SERVING_VERSION="0.22.0"
```

Comment:
And L45 pls, I can't make a suggestion for it :)
https://github.com/knative-sandbox/kn-plugin-source-kafka/blob/main/test/e2e-tests.sh#L45

Reply:
thx! Should be fixed now